sklearn/ensemble/weight_boosting.py

```diff
@@ -770,7 +773,6 @@ def predict_proba(self, X):
         normalizer = proba.sum(axis=1)[:, np.newaxis]
         normalizer[normalizer == 0.0] = 1.0
         proba /= normalizer
-
```

Comment:
Good practice: do not change code if there is not a good reason to. In this case can you please put the newline where it was?